src/pages/office/components/d255/d255.ts

```diff
@@ -34,6 +48,18 @@ export class D255Component implements OnChanges {
     }
   }
 
+  getD255OrDefault(): string | null {
+    if (this.d255 === null) {
+      if (this.outcomeBehaviourProvider.hasDefault(this.outcome, 'd255')) {
```

Comment:
magic string

Reply:
removed magic strings. will address magic strings in other components